hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/DatanodeExecutor.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.mesos.hdfs.executor;
+
+import com.google.inject.Singleton;
```

Comment:
I'm not convinced we need a separate executor for JN's and DN's since most of the logic will be the same. I'd like to avoid duplicated logic.